references.md

```diff
@@ -0,0 +1,51 @@
+## Feature References
+
+A list of every a11y concern Checka11y.css will check for and highlight with links to relevant guidelines or techniques:
+
+- [Missing `alt` attributes on images](https://www.w3.org/TR/UNDERSTANDING-WCAG20/text-equiv-all.html)
```

Comment:
Rather than wrapping the whole thing as a link, I think it would look cleaner seeing a "Read about this here" or similar link that sites next to the feature text.